tortoise/indexes.py

```diff
@@ -40,24 +40,17 @@ def __init__(
 
     def get_sql(self, schema_generator: "BaseSchemaGenerator", model: "Type[Model]", safe: bool):
         if self.fields:
-            return self.INDEX_CREATE_TEMPLATE.format(
-                exists="IF NOT EXISTS " if safe else "",
-                index_name=schema_generator.quote(
-                    self.name or schema_generator._generate_index_name("idx", model, self.fields)
-                ),
-                index_type=f" {self.INDEX_TYPE} ",
-                table_name=schema_generator.quote(model._meta.db_table),
-                fields=", ".join(schema_generator.quote(f) for f in self.fields),
-                extra=self.extra,
-            )
+            fields = ", ".join(schema_generator.quote(f) for f in self.fields)
+        else:
+            expressions = [f"({expression.get_sql()})" for expression in self.expressions]
+            fields = ", ".join(expressions)
 
-        expressions = [f"({expression.get_sql()})" for expression in self.expressions]
         return self.INDEX_CREATE_TEMPLATE.format(
             exists="IF NOT EXISTS " if safe else "",
-            index_name=self.index_name(schema_generator, model),
```

Comment:
here the method `index_name` is called, but for the first case the code of `index_name` is used instead of calling the method.